### DISCUSSION TITLE
Plot_model() strange output

### DISCUSSION THREAD

**Discussion by leodemachado** [pycaret/pycaret#1579] [Q&A]
Friends, 
when i run plot_model function, I get a different and strange result than I was supposed to get
This is how I am getting?
![Screenshot at 2021-09-16 12-02-11](https://user-images.githubusercontent.com/36859732/133662558-c30666ba-10f1-433e-8893-55d823700c3a.png)
.
This is how I was supposed to get:

![Screenshot at 2021-09-16 12-08-03](https://user-images.githubusercontent.com/36859732/133662894-821a206d-d083-48c2-9b45-d5706f68d2ac.png)
t:

Any help is appreciated.
Thanks.

**Comment by moezali1** [comment]
@leodemachado Doesn't seem strange to me at all. It seems like you have a perfect model in the first output. I can see AUC = 1.

**Reply by leodemachado** [reply]
The AUC shouldn't exactly 1, I guess... But any way, I meant the axis limits... I would like to increase it in order to see all the curves completely. Is it possible?
